lambda.js

```diff
@@ -1,16 +1,33 @@
 const server = require('restana')();
-const app = require('next')({ dev: process.env.ENV === 'dev' });
+const app = require('next')({ dev: process.env.ENV === 'dev', quiet: false });
 const files = require('serve-static');
 const path = require('path');
+const cookieParser = require('cookie-parser');
 const handle = app.getRequestHandler();
+const CheckAuth = require('./lib/use-cases/check-auth');
 
+server.use(cookieParser());
 server.use(files(path.join(__dirname, 'build')));
 server.use(files(path.join(__dirname, 'public')));
 server.use(async (req, res, next) => {
   await app.prepare();
   next();
 });
 
-server.get('*', (req, res) => handle(req, res));
+server.all('/api/*', (req, res) => handle(req, res));
 
-module.exports.handler = require('serverless-http')(server);
+server.use((req, res, next) => {
+  const checkAuth = new CheckAuth({
+    allowedGroups: process.env.allowedGroups.split(","),
+    jwt: require('jsonwebtoken')
```

Comment:
Any reason this `require` is inline but the others are declared at the top of the file? This will always run so we're not getting any lazy-loading benefit from the inline require.